tools/segments/src/index.js

```diff
@@ -1,96 +1,53 @@
+/**
+ * Copyright 2025 Adobe
+ * All Rights Reserved.
+ */
 import React from 'react';
 import * as ReactDOM from 'react-dom';
 
-import Picker from './picker.js';
-import customerSegmentsQuery from './queries/segments.graphql.js';
+import Picker from './picker';
+import getCustomerSegments from './api/segments.graphql';
+import getCatalogRules from './api/catalogrules.graphql';
+import getCartRules from './api/cartrules.graphql';
+import getCustomerGroups from "./api/customergroups.graphql";
 
 import './styles.css';
 
 /**
  * Object containing all configuration files that should be exposed in the picker.
  */
 const configFiles = {
```

Comment:
Constant can be removed. The environment names `['prod', 'stage', 'dev']` can be specified in the picker.js for switcher rendering specifically

Reply:
Refactored